fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/scalar/ScalarOperator.java

```diff
@@ -38,6 +38,9 @@ public abstract class ScalarOperator implements Cloneable {
     // from JoinNode.
     protected boolean isRedundant = false;
 
+    // whether the ScalarOperator is pushdown from equivalence derivation
+    protected boolean isPushdown = false;
```

Comment:
can we use `isRedundant`, or merge with this?

```
    // Check weather the scalar operator is redundant which will not affect the final
    // if it's not considered. eg, `IsNullPredicateOperator` which is pushed down
    // from JoinNode.
    protected boolean isRedundant = false;
```

Reply:
seems not the same purpose, maybe we can discuss it.